ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1587,8 +1591,7 @@ class PartCentredForceHolder {
     // The only timing that satisfies these constraints is BetterLateThanNever
     // in LateUpdate.
     string main_vessel_guid = PredictedVessel()?.id.ToString();
-    if (MapView.MapIsEnabled && main_vessel_guid != null &&
-        PluginRunning() && plugin_.HasVessel(main_vessel_guid)) {
+    if (MapView.MapIsEnabled && main_vessel_guid != null && PluginRunning()) {
```

Comment:
`PredictedVessel` requires `PlugginRunning`, but we are only testing that after calling it; either it was, and remains, useless here, or this is a fatal failure.

Add `PluginRunning`  to the conjunction in `PredictedVessel`.

Reply:
Done.